share/completions/keybase.fish

```diff
@@ -40,87 +41,81 @@ set -l keybase_fs_sync disable enable h help show
-complete -c keybase -f -n "$ends keybase" -l gpgdir
-complete -c keybase -f -n "$ends keybase" -l gui-config-file
-complete -c keybase -f -n "$ends keybase" -l help
-complete -c keybase -f -n "$ends keybase" -s h -d --help
-complete -c keybase -f -n "$ends keybase" -l home
-complete -c keybase -f -n "$ends keybase" -s H -d --home
-complete -c keybase -f -n "$ends keybase" -l leveldb-num-files
-complete -c keybase -f -n "$ends keybase" -l local-rpc-debug-unsafe
-complete -c keybase -f -n "$ends keybase" -l log-file
-complete -c keybase -f -n "$ends keybase" -l log-format
-complete -c keybase -f -n "$ends keybase" -l log-prefix
-complete -c keybase -f -n "$ends keybase" -l merkle-kids
-complete -c keybase -f -n "$ends keybase" -l no-auto-fork
-complete -c keybase -f -n "$ends keybase" -s F -d --no-auto-fork
-complete -c keybase -f -n "$ends keybase" -l no-debug
-complete -c keybase -f -n "$ends keybase" -l paramproof-kit
-complete -c keybase -f -n "$ends keybase" -l pgpdir
-complete -c keybase -f -n "$ends keybase" -l pid-file
-complete -c keybase -f -n "$ends keybase" -l pinentry
-complete -c keybase -f -n "$ends keybase" -l proof-cache-size
-complete -c keybase -f -n "$ends keybase" -l prove-bypass
-complete -c keybase -f -n "$ends keybase" -l proxy
-complete -c keybase -f -n "$ends keybase" -l proxy-type
-complete -c keybase -f -n "$ends keybase" -l push-disabled
-complete -c keybase -f -n "$ends keybase" -l push-save-interval
-complete -c keybase -f -n "$ends keybase" -l push-server-uri
-complete -c keybase -f -n "$ends keybase" -l pvl-kit
-complete -c keybase -f -n "$ends keybase" -l read-deleted-sigchain
-complete -c keybase -f -n "$ends keybase" -l remember-passphrase
-complete -c keybase -f -n "$ends keybase" -l run-mode
-complete -c keybase -f -n "$ends keybase" -l scraper-timeout
-complete -c keybase -f -n "$ends keybase" -l secret-keyring
-complete -c keybase -f -n "$ends keybase" -l server
-complete -c keybase -f -n "$ends keybase" -s s -d --server
-complete -c keybase -f -n "$ends keybase" -l session-file
-complete -c keybase -f -n "$ends keybase" -l slow-gregor-conn
-complete -c keybase -f -n "$ends keybase" -l socket-file
-complete -c keybase -f -n "$ends keybase" -l standalone
-complete -c keybase -f -n "$ends keybase" -l timers
-complete -c keybase -f -n "$ends keybase" -l tor-hidden-address
-complete -c keybase -f -n "$ends keybase" -l tor-mode
-complete -c keybase -f -n "$ends keybase" -l tor-proxy
-complete -c keybase -f -n "$ends keybase" -l updater-config-file
-complete -c keybase -f -n "$ends keybase" -l use-default-log-file
-complete -c keybase -f -n "$ends keybase" -l use-root-config-file
-complete -c keybase -f -n "$ends keybase" -l user-cache-size
-complete -c keybase -f -n "$ends keybase" -l vdebug
-complete -c keybase -f -n "$ends keybase" -l version
-complete -c keybase -f -n "$ends keybase" -s v -d --version
+complete -c keybase -f -n "$ends keybase" -l api-dump-unsafe                   -d ""
```

Comment:
Please don't add useless empty descriptions.

Reply:
Keybase completions are still WIP and not completed. Should I remove the empty placeholders for now?

Reply:
If you're still working on them, leave it.